### DISCUSSION TITLE
This repo will not be archived

### DISCUSSION THREAD

**Discussion by leahanderson1** [EaglerReborn/support#4] [Announcements]
since the new repo is up, I will be archiving this repo in 1 week.

**Comment by ZXMushroom63** [comment]
since the new repo is down, this repo should probably not be archived.

**Comment by FaZeS3iped** [comment]
any updates on the dmca takedown?

**Comment by FaZeS3iped** [comment]
this project was exported to my flash drive as a html file and everything works except the java servers

**Comment by FaZeS3iped** [comment]
Says: sorry query isnt supported yet for java

**Reply by leahanderson1** [reply]
it still lets you join, thats just the message i coded in because query does not work

**Reply by FaZeS3iped** [reply]
Any websites for Eagler-Reborn? I'm running on a HTML file rn.

**Reply by FaZeS3iped** [reply]
Ya every java server I join says connection lost after trying to join

**Reply by wxnnvs** [reply]
> Ya every java server I join says connection lost after trying to join

Ah i see, u need cracked java or you'll have to ise another relay
